sizefashion/src/components/Filters/VendorFilters.tsx

```diff
@@ -0,0 +1,36 @@
+import { useFilterContext } from '../../context/filter_context'
+import { getUniqueValues } from '../../utils/helpers'
+
+export const VendorFilters = () => {
+  const {
+    updateFilters,
+    allProducts,
+    filters: { vendor },
+  } = useFilterContext()
+  const uniqueVendors = getUniqueValues(allProducts, 'vendor')
+  return (
+    <div>
+      <label className='block text-black text-md font-bold my-2'>Marcas</label>
+      <div className='overflow-x-hidden overflow-y-auto max-h-96'>
+        {uniqueVendors.map((uniqueVendor) => {
+          if (typeof uniqueVendor === 'string') {
```

Comment:
@otaaviostein based on what I saw on your code, `uniqueVendor` is already a literal string. Wouldn't it work to just remove the if statement and let the loop run by itself?